api/discord-client.js

```diff
@@ -0,0 +1,12 @@
+import { Client, Intents } from 'discord.js';
+
+export let isClientReady = false;
+
+const discordClient = new Client({ intents: [Intents.FLAGS.GUILDS] });
```

Comment:
Not sure how the client is receiving the auth token. I don't remember if we had that, but an assert would be nice (if it's an environment variable).